nameko_sqlalchemy/pytest_fixtures.py

```diff
@@ -140,3 +142,23 @@ def db_session(db_connection, model_base):
 
     db_session.commit()
     db_session.close()
+
+
+@pytest.yield_fixture
+def database(db_connection, model_base):
+
+    def reset(database):
+        with database.get_session() as session:
+            for table in reversed(model_base.metadata.sorted_tables):
+                session.execute(table.delete())
+
+    database = DatabaseWrapper(
+        sessionmaker(bind=db_connection, class_=Session))
+
+    reset(database)
```

Comment:
Weird to be deleting the tables before the `yield`. Is this intentional?

Reply:
No specific need for doing it. Removed.)

Reply:
Ahh, `table.delete` actually truncates and doesn't drop the schema. That's why it wasn't completely broken before.

So yes, no need to truncate already-empty tables ;)